great_expectations/execution_engine/split_and_sample/sqlalchemy_data_sampler.py

```diff
@@ -15,6 +15,7 @@
     sa = None
 
 try:
+    import sqlalchemy as sa
```

Comment:
Looks like this is a duplicate (see above on line 13?)

Reply:
got it. thank you for catching